model/converter/json/to_domain.go

```diff
@@ -184,6 +185,9 @@ func (td toDomain) convertLogs(logs []json.Log) ([]model.Log, error) {
 }
 
 func (td toDomain) convertProcess(process *json.Process) (*model.Process, error) {
+	if process == nil {
```

Comment:
fyi it happened when i was running a basic test with a json model span with just traceID bc i didnt need all the parts